index.html

```diff
@@ -770,8 +771,9 @@
 
 reader.addEventListener("reading", event => {
   for (let record of event.message.records) {
-    let article =/[aeio]/.test(record.data().title) ? "an" : "a";
-    console.log(`${record.data().name} is ${article} ${record.data().title}`);
+    let json = record.toJSON();
```

Comment:
if you never modify the value, use const not let

Reply:
Thanks, fixed.